olm-catalog/serverless-operator/project.yaml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
I'm wondering if this `project.yaml` should be moved to the top-level directory. I think it affects the whole repo.

Reply:
It was at top level. I moved it into `olm-catalog/serverless-operator` so it will be packaged inside the image container. Then we get access to those variables via image.